community/lld/sources

```diff
@@ -0,0 +1,2 @@
+https://github.com/llvm/llvm-project/releases/download/llvmorg-11.1.0/llvm-11.1.0.src.tar.xz
```

Comment:
I was under the impression the LLVM Project was only going to be distributing `llvm` as a whole, and not separating out the projects. But I guess this is incorrect?
Either way, you shouldn't need both `llvm` *and* `lld`. Just `lld` should suffice.

Reply:
I think I was confusing myself by the LLVM documentation, but you are correct! I've made the necessary changes and am pushing now.